### PR TITLE
Improve the documentation of `Filesystem::mirror()`

### PR DESCRIPTION
**Pull request by mnapoli**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      |no
| New feature?  |no 
| BC breaks?    |no
| Deprecations? |no 
| Tests pass?   | ?
| Fixed tickets | 
| License       | MIT
| Doc PR        | 

I always end up reading the code of the method to understand what really happens with the default parameters. Hopefully this will clear things up for future me and others.

### CODE REVIEW

File: src/Symfony/Component/Filesystem/Filesystem.php [hunk 1]
```diff
@@ -524,13 +524,18 @@ public function makePathRelative($endPath, $startPath)
     /**
      * Mirrors a directory to another.
      *
+     * All files and directories in the origin directory will be copied into the target directory. By default:
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Filesystem/Filesystem.php]: > All files and directories[...]
not true if $iterator is provided, isn't it?

File: src/Symfony/Component/Filesystem/Filesystem.php [hunk 2]
```diff
@@ -524,13 +524,18 @@ public function makePathRelative($endPath, $startPath)
...
+     *
+     * - existing files in the target directory will be overwritten, except if they are newer (see the `override` option)
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Filesystem/Filesystem.php]: the bullet misses a space before (one for the indentation of the docblock, one for the indentation of the bullet)

**Review by nicolas-grekas — CHANGES_REQUESTED** [review]
should be merged on 2.7
@mnapoli can you rebase/retarget if you don't mind please?

**Comment by mnapoli** [comment]
No problem, I've opened #26215 instead (I'm doing these changes in the browser, that's just simpler for me).

File: src/Symfony/Component/Filesystem/Filesystem.php (hunk 2 above)
Comment (mnapoli) [reply src/Symfony/Component/Filesystem/Filesystem.php]: Fixed in #26215

File: src/Symfony/Component/Filesystem/Filesystem.php (hunk 1 above)
Comment (mnapoli) [reply src/Symfony/Component/Filesystem/Filesystem.php]: Right, I've rephrased a bit in #26215 and expanded the documentation of the `$iterator` parameter.

**Comment by mnapoli** [comment]
Also just to be sure should I open all phpdoc changes against 2.7?

**Comment by nicolas-grekas** [comment]
> Also just to be sure should I open all phpdoc changes against 2.7?

yes please, the lowest branch where they apply, like bugs

**Pull request closed without merging**